dronegen/mac.go

```diff
@@ -541,6 +541,8 @@ func darwinBuildCommands(toolchainConfig toolchainConfig, artifactConfig darwinA
 			// available.
 			// https://www.electron.build/code-signing
 			`export CSC_NAME=0FFD3E3413AB4C599C53FBB1D8CA690915E33D83`,
+
+			`export DEBUG=electron-*`,
```

Comment:
```suggestion
			`export DEBUG="electron-*"`,
```

Should we quote this? I'm not sure if shell wildcard expansion will kick in otherwise.

Reply:
Ah, yeah I'm not sure either and it worked fine without quotes for me but it's a good practice anyway, I'll add it.